### PR TITLE
Updated redux dependency version

### PR DESCRIPTION
**Pull request by juangaspar**

### CODE REVIEW

**Comment by Cretezy** [comment]
Thank you, forgot about this. It doesn't seem like there is any breaking changes in Redux 4.0, however have you tested this?

File: packages/redux_persist/pubspec.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ environment:
   sdk: ">=2.0.0-dev <3.0.0"
 
 dependencies:
-  redux: "^3.0.0"
+  redux: "^4.0.0"
```
Comment (analogic) [inline packages/redux_persist/pubspec.yaml]: I think _">=3.0.0 <5.0.0"_ would be better

**Comment by analogic** [comment]
> Thank you, forgot about this. It doesn't seem like there is any breaking changes in Redux 4.0, however have you tested this?

Yep, it is working ok

**Comment by juangaspar** [comment]
> Thank you, forgot about this. It doesn't seem like there is any breaking changes in Redux 4.0, however have you tested this?

Yes, at least in my tests, its working fine

**Review by Cretezy — APPROVED** [review]

**Comment by Cretezy** [comment]
Will include in next release (soon)

Thank you!

**Pull request merged**